src/SpectatorView.Unity/Assets/SpectatorView/Scripts/MarkerDetection/ArUco/HoloLens/ArUcoMarkerDetector.cs

```diff
@@ -116,11 +122,28 @@ private void Update()
         {
             if (_detecting)
             {
+#if UNITY_EDITOR
+                int markerCount = UnityCompositorInterface.GetLatestArUcoMarkerCount();

```

Comment:
the legibility of this code seems to suffer based on the editor vs non editor logic calls. Is there a major motivation behind not making a different ar uco marker detector that is specific to the compositor and implements IMarkerDetector? It seems like we don't use any of the same native logic calls? Someone may try and use this in the editor unrelated to the compositor if they see these #if UNITY_EDITOR stubs.